### PR TITLE
 Fix #6962 - Long press on search icon should not show any options

### LINKED ISSUE #6962: Longpress on search icon will open long press options via reload button
## Steps to reproduce
1. Open a new tab
2. Long press on the Search icon
### Expected behavior
- No options should be visible.
### Actual behavior
- "Request Desktop Site" and "Reload without Tracking Protection" options are revealed
### Device & build information
* Device: iPhone Xr
* Build version: main 010453407c58

Screen Shot 2020-07-14 at 13 17 17

Attachments:

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

File: Client/Frontend/Browser/TabToolbar.swift [hunk 1]
```diff
@@ -195,6 +195,7 @@ open class TabToolbarHelper: NSObject {
     }
 
     func didLongPressStopReload(_ recognizer: UILongPressGestureRecognizer) {
+        guard !isSearch else { return }
```
Comment (nbhasin2) [inline Client/Frontend/Browser/TabToolbar.swift:198]: Added a guard to not trigger long press for when 🔍 icon is being shown.

**Comment by aabbccddff9961** [comment]
+1

File: Client/Frontend/Browser/TabToolbar.swift (hunk 1 above)
Comment (aabbccddff9961) [reply Client/Frontend/Browser/TabToolbar.swift:198]: Added a guard to not trigger long press for when 🔍 icon is being shown.

Comment (aabbccddff9961) [reply Client/Frontend/Browser/TabToolbar.swift:198]: Added a guard to not trigger long press for when 🔍 icon is being shown.

**Pull request merged**